content/it/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,67 @@
+
+Per verificare questa connessione, si utilizzi il parametro `--kubelet-certificate-authority` al fine di fornire all'apiserver un insieme di certificati radice da utilizzare per verificare il
+il certificato servito dal _kubelet_.
+
+Se questo non è possibile, si usi un [tunnel SSH](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) tra l'apiserver e il _kubelet_, se richiesto, per evitare il collegamento su una rete non fidata o pubblica.
+
+In fine, l'[autenticazione e/o l'autorizzazione del kubelet](/docs/admin/kubelet-authentication-authorization/) dovrebbe essere abilitate per proteggere le API esposte dal _kubelet_.
+
+### Dall'apiserver ai nodi, _pod_, e _servizi_{#dal-apiserver-ai-nodi-pod-servizi}
+
+Le connessioni dall'apiserver verso un nodo, _pod_ o servizio avvengono in modalità predefinita su semplice connessione HTTP e quindi non sono né autenticate né criptata. Queste connessioni possono essere eseguite su una connessione HTTPS sicura mediante il prefisso `https:` al nodo, pod o nome del servizio nell'URL dell'API, ma non valideranno il certificato fornito dall'endpoint HTTPS né forniranno le credenziali del client così anche se la connessione verrà criptata, non fornirà alcuna garanzia di integrità. **Non è attualmente sicuro** eseguire queste connessioni  su reti non fidate e/o pubbliche.
+
+### I tunnel SSH
+
+Kubernetes supporta i _tunnel_ SSH per proteggere la comunicazione tra il piano di controllo e i nodi. In questa configurazione, l'apiserver inizializza un tunnel SSH con ciascun nodo del cluster (collegandosi al server SSH in ascolto sulla porta 22) e fa passare tutto il traffico verso il _kubelet_, il nodo, il _pod_, o il servizio attraverso questo tunnel. Questo tunnel assicura che il traffico non sia esposto al di fuori della rete su cui sono in esecuzioni i vari nodi.
+
+I tunnel SSH sono al momento deprecati ovvero non dovrebbero essere utilizzati a meno che ci siano delle esigenze particolari. Il servizio `Konnectivity` è pensato per rimpiazzare questo canale di comunicazione.
+
+### Il servizio _Konnectivity_
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+Come rimpiazzo dei tunnel SSH, il servizio `Konnectivity` fornisce un proxy a livello TCP per la comunicazione tra il piano di controllo e il cluster. Il servizio `Konnectivity` consiste in due parti: il `Konnectivity` server e gli agenti `Konnectivity`, in esecuzione rispettivamente sul piano di controllo e sui vari nodi.  Gli agenti `Konnectivity` inizializzano le connessioni verso il server `Konnectivity` e mantengono le connessioni di rete. Una volta abilitato il servizio `Konnectivity`, tutto il traffico tra il piano di controllo e i nodi passa attraverso queste connessioni.
+
+Si usi il [tutorial per il servizio `Konnectivity`](/docs/tasks/extend-kubernetes/setup-konnectivity/) per configurare il servizio `Konnectivity` all'interno del cluster
```

Comment:
I recommend against writing Konnectivity in backticks - that makes it look like a command line you might invoke.

Reply:
backticks removed